lib/Page.js

```diff
@@ -23,6 +23,7 @@ function Page(pageConfig) {
   this.baseUrl = pageConfig.baseUrl;
   this.asset = pageConfig.asset;
   this.baseUrlMap = pageConfig.baseUrlMap;
+  this.baseUrlToDefinedVariablesMap = pageConfig.baseUrlToDefinedVariablesMap;
```

Comment:
If this has a long name, it should really be "baseUrlTo`User`DefinedVariablesMap".
Anyway, I believe the key isn't `baseUrl` but it's the base directory of the (sub)site.

How about `userDefinedVariablesMap`? (The method name too.)